src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/CustomCaConfigurationTest.java

```diff
@@ -107,14 +110,17 @@ void beforeEach() {
 
         certificateManager = new CertificateManager(
                 certificateStore, mockConnectivityInformation,
-                mockCertExpiryMonitor, mockShadowMonitor, Clock.systemUTC(), clientFactoryMock, securityServiceMock);
+                mockCertExpiryMonitor, mockShadowMonitor, Clock.systemUTC(), clientFactoryMock);
 
+        topics = Topics.of(new Context(), CLIENT_DEVICES_AUTH_SERVICE_NAME, null);
```

Comment:
context must always be closed at the end of a test. It starts a thread and we don't want that to run forever. Make sure you always shutdown any context you create.